Content.Shared/Alert/SharedAlertsComponent.cs

```diff
@@ -180,12 +183,12 @@ public AlertsComponentState(Dictionary<AlertKey, AlertState> alerts)
     [Serializable, NetSerializable]
     public class ClickAlertMessage : ComponentMessage
     {
-        public readonly AlertType AlertType;
+        public readonly AlertType Type;
```

Comment:
Not directly related to the bug, but I couldn't leave `AlertType AlertType` alone. And yeah, `Type` isn't a great name either, but it's still better and I CBF thinking of another.

Reply:
You should probably leave discussion for Discord.

Reply:
naming is fine